arduino-ide-extension/src/browser/contributions/open-sketch.ts

```diff
@@ -82,7 +82,10 @@ export class OpenSketch extends SketchContribution {
   }
 
   private async selectSketch(): Promise<Sketch | undefined> {
-    const config = await this.configService.getConfiguration();
+    const config = this.configService.tryGetConfig();
+    if (!config) {
+      return undefined;
```

Comment:
Incorrect. Opening a sketch should work even if the sketchbook path is invalid/not set.

Reply:
https://github.com/arduino/arduino-ide/pull/1774/commits/707f59dbc9594738655f4b2d2d4c03e317c3e384